Mathlib/CategoryTheory/Limits/Shapes/Multiequalizer.lean

```diff
@@ -752,8 +752,6 @@ theorem ιPi_π (a) : ιPi I ≫ Pi.π I.left a = ι I a := by
   rw [ιPi, Category.assoc, ← Iso.eq_inv_comp, isoEqualizer]
   simp
 
-instance : Mono (ιPi I) := mono_comp _ _
```

Comment:
I'm gonna guess that this still needs to be an instance since instance resolution can't look through the definition otherwise. I might be wrong though...

Reply:
Oh yes, good point!